sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollerFlux.java

```diff
@@ -139,6 +155,80 @@ public PollerFlux(Duration pollInterval,
             true);
     }
 
+    /**
+     * Creates PollerFlux.
+     *
+     * This create method uses a {@link PollingStrategy} to poll the status of a long running operation after the
+     * activation operation is invoked. See {@link PollingStrategy} for more details of known polling strategies
+     * and how to create a custom strategy.
+     *
+     * @param pollInterval the polling interval
+     * @param activationOperation the activation operation to activate (start) the long running operation.
+     *     This operation will be invoked at most once across all subscriptions. This parameter is required.
+     *     If there is no specific activation work to be done then invocation should return Mono.empty(),
+     *     this operation will be called with a new {@link PollingContext}.
+     * @param strategy a known strategy for polling a long running operation in Azure
+     * @param resultType the {@link Type} of the final result object to deserialize into, or BinaryData if raw response
+     *                   body should be kept. This should match the generic parameter {@link U}.
+     * @param httpPipeline an instance of {@link HttpPipeline} to send requests with
+     * @param context additional metadata to pass along with the request
+     * @param <U> The type of the final result of long running operation
+     * @return PollerFlux
+     */
+    public static <U> PollerFlux<BinaryData, U>
+        create(Duration pollInterval,
+               Supplier<Mono<? extends Response<?>>> activationOperation,
+               PollingStrategy strategy,
+               Type resultType,
+               HttpPipeline httpPipeline,
+               Context context) {
+        return create(
+            pollInterval,
+            ctx -> activationOperation.get()
+                .flatMap(r -> {
+                    if (!strategy.canPoll(r)) {
+                        return Mono.error(new IllegalStateException("Cannot poll with strategy " + strategy));
+                    }
+                    return strategy.onActivationResponse(r, ctx)
+                        .map(status -> {
+                            if (r.getValue() instanceof BinaryData) {
+                                return new PollResponse<>(status, (BinaryData) r.getValue());
```

Comment:
Does data-plane provide the `Duration retryAfter` header?

Reply:
It's handled in the individual polling strategies: https://github.com/jianghaolu/azure-sdk-for-java/blob/feature/jianghao/lro/sdk/core/azure-core/src/main/java/com/azure/core/util/polling/OperationResourcePollingStrategy.java#L120